### PR TITLE
Polish up work on #108 on the irispy/time_dependent_response branch

### LINKED ISSUE #27: Time-Dependent IRIS Effective Areas
Enable time-dependent effective areas to be derived using get_iris_response() in iris_tools in the same way that it is done in SSW.  See iris_get_response in SSW, a version of which can be found here: https://sohowww.nascom.nasa.gov/solarsoft/iris/idl/lmsal/util/iris_get_response.pro

### PR DESCRIPTION
**Pull request by kakirastern**
Fixes #27. 

Goals
=====
- [x] Default to version 4 if neither response `version` nor `file` has been explicitly passed to the `get_iris_response` method.
- [x] Convert the `time_obs` variable from a numpy.array to an astropy.time.Time object.

### CODE REVIEW

**Comment by DanRyanIrish** [comment]
Hi @krisastern.  I would suggest that Goal 3 be it's own PR.  That way this one can be merged more quickly and is easier to review.

**Comment by kakirastern** [comment]
Sure @DanRyanIrish 
I will make the changes soon.

**Comment by kakirastern** [comment]
Hi @DanRyanIrish! Changes made, and the PR is ready for a review.

File: irispy/iris_tools.py [hunk 1]
```diff
@@ -82,8 +82,8 @@ def get_iris_response(time_obs=None, pre_launch=False, response_file=None, respo
 
     Parameters
     ----------
-    time_obs: a `numpy.array` of floats, as a kwarg, valid for version > 2
-        Observation times of the datapoints. Must be in the format of the output of time_obs = parse_time('2013-09-03', format='utime'), which yields 1094169600.0 seconds. The argument time_obs is ignored for versions 1 and 2.
+    time_obs: an `astropy.time.Time` object, as a kwarg, valid for version > 2
+        Observation times of the datapoints. Must be in the format of , e.g., time_obs = parse_time('2013-09-03', format='utime'), which yields 1094169600.0 seconds in value. The argument time_obs is ignored for versions 1 and 2.
```
Comment (DanRyanIrish) [inline irispy/iris_tools.py]: PEP 8 compliance: Can you break this into multiple lines no line hits the 100 character mark?

**Comment by DanRyanIrish** [comment]
Great.  Once it passes the CI tests, I'll approve and merge :)

**Pull request merged**